packages/react/src/components/ContextMenu/ContextMenu-story.js

```diff
@@ -37,6 +54,43 @@ const Story = (items) => {
   );
 };
 
+const Text = () => (
+  <div style={{ width: '40rem' }}>
+    <h2>Right click anywere in the story frame</h2>
+    <br />
+    <p>
+      The <CodeSnippet type="inline">{`<Menu>`}</CodeSnippet> props are set
+      using the hook{' '}
+      <CodeSnippet type="inline">{`useContextMenu()`}</CodeSnippet>. This
+      determines the position the menu should have on right click and handles
+      opening. Props that the{' '}
+      <CodeSnippet type="inline">{`useContextMenu()`}</CodeSnippet>
+      hook does not set and can be configured by the user are:
+      <UnorderedList style={{ paddingLeft: '2rem', margin: '2rem 0' }}>
+        <ListItem>size</ListItem>
+        <ListItem>onClose</ListItem>
+        <ListItem>className</ListItem>
+        <ListItem>id</ListItem>
+        <ListItem>target</ListItem>
+      </UnorderedList>
+    </p>
+    <p>
+      The
+      <CodeSnippet type="inline">{`<MenuGroup>`}</CodeSnippet> and
+      <CodeSnippet type="inline">{`<MenuItem`}</CodeSnippet>
+      components accept children items for nested menus, although the{' '}
+      <CodeSnippet type="inline">{`<MenuItem>`}</CodeSnippet> component can also
+      be used as a stand alone item. The other types of menu items ({' '}
+      <CodeSnippet type="inline">{`<MenuDivider>`}</CodeSnippet>,
+      <CodeSnippet type="inline">{`<MenuSelectableItem>`}</CodeSnippet>,
+      <CodeSnippet type="inline">{`<MenuRadioGroup>`}</CodeSnippet>) do not
+      accept children. The{' '}
+      <CodeSnippet type="inline">{`<MenuRadioGroup>`}</CodeSnippet> accepts an
+      array of items to display as a group of single choice selection.
+    </p>
+  </div>
+);
+
 export const _ContextMenu = () =>
   Story([
```

Comment:
Could we refactor these two stories (ContextMenu, MultipleGroups) to follow the normal pattern instead of passing an array of items to a function? The story source panel doesn't even load for me on these stories.

I think using a function to generate the items via `buildMenu(items)` is fine, it's just calling the `Story` function that seems to be gunking things up.

Reply:
@tay1orjones yeah, it's not great. I can update these 👍🏽